jenkins/ci.suse.de/templates/cloud-mkcloud-job-upgrade-template.yaml

```diff
@@ -1,7 +1,7 @@
 - job-template:
     name: 'cloud-mkcloud{version}-job-upgrade-{arch}'
     node: cloud-trigger
-    disabled: '{obj:disabled}'
```

Comment:
This has no effect, the cloud6 upgrade job remains **enabled** because it was deleted from JJB with the above change.

Is this wanted? The title of PR indicates that it is not wanted.

My recommendation:
1. Undelete the cloud6 upgrade job in cloud-mkcloud6.yaml
2. keep the template disabled (as it is here), this will keep the job in Jenkins -- then disabled